ports/catwarrior/Cat Warrior.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+# Pm
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+source $controlfolder/tasksetter
+
+# Variables
+GAMEDIR="/$directory/ports/catwarrior"
+cd "$GAMEDIR/gamedata"
+
+# Setup mono
+monodir="$HOME/mono"
+monofile="$controlfolder/libs/mono-6.12.0.122-aarch64.squashfs"
+$ESUDO mkdir -p "$monodir"
+$ESUDO umount "$monofile" || true
+$ESUDO mount "$monofile" "$monodir"
+
+# Setup savedir linux
+$ESUDO rm -rf ~/.local/share/tiny_slash
+mkdir -p ~/.local/share
+mkdir -p "$GAMEDIR/savedata"
+ln -sfv "$GAMEDIR/savedata" ~/.local/share/tiny_slash
+bind_directories "$HOME/.local/share/tiny_slash" "$GAMEDIR/savedata"
+
+# Remove all the dependencies in favour of system libs - e.g. the included 
+rm -f System*.dll mscorlib.dll FNA.dll Mono.*.dll
+
+# Setup path and other environment variables
+export MONO_PATH="$GAMEDIR/dlls"
+export LD_LIBRARY_PATH="$GAMEDIR/libs:/usr/config/emuelec/lib32:/usr/lib32:$LD_LIBRARY_PATH"
+export PATH="$monodir/bin:$PATH"
+
+export FNA3D_FORCE_DRIVER=OpenGL
+export FNA3D_OPENGL_FORCE_ES3=1
+export FNA3D_OPENGL_FORCE_VBO_DISCARD=1
+
+$GPTOKEYB "mono" &
+pm_platform_helper "mono"
+$TASKSET mono tiny_slash.exe 2>&1 | tee /dev/tty0 $GAMEDIR/log.txt
```

Comment:
old logging?

Reply:
Yeah this is what johnny used on Celeste. I'll update this later today.